app/helpers/pair_requests_helper.rb

```diff
@@ -1,2 +1,12 @@
 module PairRequestsHelper
+  def format_request_date(pair_request)
```

Comment:
fine with this for now but if we start to notice repeated behavior we can refactor then.